[DEVEX-300] Switch to Maven Publish

danielkatfavor opened the pull request:
Ticket
------------------------------------------------------------------------
[DEVEX-300]

Context
------------------------------------------------------------------------
This change replaces the current publishing mechanism in the forked
version of the IndicatorFastScroll codebase so that developers can
publish to our internal Favor Delivery packages.

Changes
------------------------------------------------------------------------
* Updated Gradle Wrapper.
* Updated AGP.
* Switched out publishing mechanism.

[DEVEX-300]: https://favorengineering.atlassian.net/browse/DEVEX-300?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

danielkatfavor committed:
[DEVEX-300] Switch to Maven Publish

Context
------------------------------------------------------------------------
This change replaces the current publishing mechanism in the forked
version of the IndicatorFastScroll codebase so that developers can
publish to our internal Favor Delivery packages.

Changes
------------------------------------------------------------------------
* Updated Gradle Wrapper.
* Updated AGP.
* Switched out publishing mechanism.

igokoro commented:
@danielkatfavor , we may also want to update `readme` to indicate that publishing a new version is all manual process - there's no automation that this repository offers.

danielkatfavor committed:
[DEVEX-300] Address PR comments

Changes
------------------------------------------------------------------------
* Updated README.
